deploy.ps1

```diff
@@ -42,14 +42,46 @@ $azureReqs = @(
     }
 )
 
+# Detect global infra
+$globalReqs = @(
+    @{
+        Name     = "Terraform init"
+        Describe = "Initialize terraform environment"
+        Test     = { Test-Path "$PSScriptRoot/tf/global/.terraform/terraform.tfstate" }
+        Set      = {
+            Set-Location -Path "tf/global"
+            terraform init
+        }
+    },
+    @{
+        Name     = "Terraform plan"
+        Describe = "Plan terraform environment"
+        Test     = { Test-Path "$OutputDir/global.plan" }
+        Set      = {
+            New-Item -Path "$OutputDir" -ItemType Directory -Force
+            terraform plan -out "$OutputDir/global.plan"
+        }
+    },
+    @{
+        Name     = "Terraform Apply"
+        Describe = "Apply Terraform plan"
+        Test     = { Test-Path "$OutputDir/global" }
+        Set      = {
+            terraform apply "$OutputDir/out.plan" | Write-Information
+            terraform output kube_config | Out-File "$OutputDir/azurek8s"
+            Set-Location $RepoRoot
+        }
+    }
+)
+
 # Provision Infra
 $tfReqs = @(
     @{
         Name     = "Terraform init"
         Describe = "Initialize terraform environment"
-        Test     = { Test-Path "$PSScriptRoot/tf/.terraform/terraform.tfstate" }
+        Test     = { Test-Path "$PSScriptRoot/tf/enclave/.terraform/terraform.tfstate" }
```

Comment:
An `enclave` is what I propose referring to our secured clusters as.  It implies security and sounds neat.